rasa/cli/x.py

```diff
@@ -220,67 +226,38 @@ def is_rasa_project_setup(project_path: Text):
     return True
 
 
-def rasa_x(args: argparse.Namespace):
-    from rasa.cli.utils import print_success, print_error, signal_handler
-    from rasa.core.utils import AvailableEndpoints
+def _validate_rasa_x_start(args: argparse.Namespace, project_path: Text):
+    if not is_rasa_x_installed():
+        print_error(
+            "Rasa X is not installed. The `rasa x` "
+            "command requires an installation of Rasa X."
+        )
+        sys.exit(1)
 
-    signal.signal(signal.SIGINT, signal_handler)
+    if args.port == args.rasa_x_port:
+        print_error(
+            "The port for Rasa X and the port of the Rasa server are the same. "
+            "We need two different ports, one to run Rasa X (e.g. delivering the UI) "
+            "and another one to run a normal Rasa server.\nPlease specify two "
+            "different ports using the arguments '--port' and '--rasa-x-port'."
+        )
+        sys.exit(1)
 
-    _configure_logging(args)
+    if not is_rasa_project_setup(project_path):
+        print_error(
+            "This directory is not a valid Rasa project. Use 'rasa init' "
+            "to create a new Rasa project or switch to a valid Rasa project "
+            "directory."
```

Comment:
can we also add a link to some docs here? (I think there might be something about the directory structure)